### PR TITLE
Update Follow on Shop Option Text

### PR DESCRIPTION
**Pull request by schaab**
### PR Summary: 

Feedback was given to update the text of the Follow on Shop feature to include some more upfront information about requirements for it be displayed.

### Why are these changes introduced?

> Display follow button for your storefront on the Shop app. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/follow-on-shop)

👆is currently displayed. When the merchant toggles the checkbox the button may or may not show up depending of the configuration of their store. It isn't clear that there are prerequisites that need to be met before the button will show up.

### What approach did you take?

Update the text to:

> To allow shoppers to follow your store on the Shop app from your storefront, Shop Pay must be enabled. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/follow-on-shop)

This alludes to the prerequisite being present.

### Visual impact on existing themes

The text in the theme editor will be updated 👇

Screenshot 2023-03-22 at 3 21 29 PM

### Testing steps/scenarios

- [ ] Navigate to theme editor and the footer section. Verify text matches screenshot above.

### Demo links

- [Editor](https://admin.shopify.com/store/jared-schaab/themes/130951086136/editor?section=sections--15956155236408__footer)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)

### CODE REVIEW

File: locales/en.default.schema.json [hunk 1]
```diff
@@ -1013,7 +1013,7 @@
         },
          "header__9": {
           "content": "Follow on Shop",
-          "info": "Display follow button for your storefront on the Shop app. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/follow-on-shop)"
+          "info": "To allow shoppers to follow your store on the Shop app from your storefront, Shop Pay must be enabled. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/follow-on-shop)"
```
Comment (LucasLacerdaUX) [inline locales/en.default.schema.json:1016]: Is `shoppers` a term we use in our content UX? I can find a few references to `customers` in our translation files and our help docs also use the same term:

image

Comment (schaab) [reply locales/en.default.schema.json:1016]: I got this text from UX, so I am going to allow them to chime in.

@bjorkeson thoughts and feelings?

Comment (bjorkeson) [reply locales/en.default.schema.json:1016]: @Michael-Webster21 What do you think about this?

**Comment by ludoboludo** [comment]
@schaab 
Since the `z-index` seem to be used specifically for the button itself and its styling, I think it would be great to apply this CSS on it so it create a new stacking context specific to the button: `isolation: isolate;` and will prevent the button from showing on top of other elements. 
That could probably be applied to parent element, maybe `<shop-follow-button` ? 

Let me know if that would be doable on your end and if yes, what would be the timeframe ? Since it's causing an issue on our themes we would want it fixed soon/by the next release which is beginning of May.

**Review by LucasLacerdaUX — APPROVED** [review]

**Comment by schaab** [comment]
Ooof, that is no good. I have created an issue https://github.com/Shopify/shop-js/issues/945 for this.

For context I am working on getting some other work out this week and likely I can pick it up starting next week.

**Pull request merged**